### PR TITLE
pagetree minor tests

### PR DESCRIPTION
**Pull request by vxsx**

### CODE REVIEW

File: cms/tests/frontend/integration/pagetree.js [hunk 1]
```diff
@@ -162,6 +163,101 @@ casper.test.begin('Correctly displays languages', function (test) {
...
+        .start()
+        .then(cms.addPage({ title: 'Homepage' }))
+        .thenOpen(globals.baseUrl)
+        .then(cms.openSideframe())
+        // switch to sideframe
+        .withFrame(0, function () {
+            casper.waitForSelector('.cms-pagetree', function () {
+                var pageId = cms.getPageId('Homepage');
+                // check that languages look correct
+
```
Comment (FinalAngel) [inline cms/tests/frontend/integration/pagetree.js]: why the space?

**Comment by FinalAngel** [comment]
LGTM

File: cms/tests/frontend/integration/pagetree.js (hunk 1 above)
Comment (vxsx) [inline cms/tests/frontend/integration/pagetree.js]: idk lol

**Comment by FinalAngel** [comment]
All this waiting :D

**Comment by vxsx** [comment]
@FinalAngel you have no idea :D

**Pull request merged**